src/Features/Gallery/Pages/BuiltIn/CollectionView/CollectionViewPage.xaml

```diff
@@ -5,14 +5,306 @@
+                                    </Grid.ColumnDefinitions>
+                                    <Image Grid.RowSpan="2" 
+                                   Source="{Binding ImageUrl}" 
+                                   Aspect="AspectFill"
+                                   HeightRequest="60" 
+                                   WidthRequest="60" />
+                                    <Label Grid.Column="1" 
+                                   Text="{Binding Name}" 
+                                   FontAttributes="Bold" />
+                                    <Label Grid.Row="1"
+                                   Grid.Column="1" 
+                                   Text="{Binding Location}"
+                                   FontAttributes="Italic" 
+                                   VerticalOptions="End" />
+                                </Grid>
+                            </SwipeView>
+                        </DataTemplate>
+                    </CollectionView.ItemTemplate>
+                </CollectionView>
+            </StackLayout>
+
+            <Label Style="{x:StaticResource DocumentSectionTitleStyle}" Text="CollectionView layouts" />
+            <Label  Text="Vertical list layout" />
+            <StackLayout Margin="20">
+                <CollectionView ItemsSource="{Binding Monkeys}">
+                    <CollectionView.ItemTemplate>
+                        <DataTemplate>
+                            <Grid Padding="10">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="Auto" />
+                                    <ColumnDefinition Width="Auto" />
+                                </Grid.ColumnDefinitions>
+                                <Image Grid.RowSpan="2" 
+                               Source="{Binding ImageUrl}" 
+                               Aspect="AspectFill"
+                               HeightRequest="60" 
+                               WidthRequest="60" />
+                                <Label Grid.Column="1" 
+                               Text="{Binding Name}" 
+                               FontAttributes="Bold" />
+                                <Label Grid.Row="1"
+                               Grid.Column="1" 
+                               Text="{Binding Location}"
+                               FontAttributes="Italic" 
+                               VerticalOptions="End" />
+                            </Grid>
+                        </DataTemplate>
+                    </CollectionView.ItemTemplate>
+                </CollectionView>
+            </StackLayout>
+            
+            <Label  Text="Vertical grid layout" />
```

Comment:
I NEED to make one CollectionView and then change the layout mode using the radiobuttons control